create-db-ddl.sql

```diff
@@ -503,12 +468,6 @@ ALTER TABLE cqc.services_id_seq OWNER TO sfcadmin;
 ALTER SEQUENCE IF EXISTS cqc.services_id_seq OWNED BY cqc.services.id;
 
 
---
--- Name: CqcLog id; Type: DEFAULT; Schema: cqc; Owner: postgres
---
-
-ALTER TABLE ONLY cqc."CqcLog" ALTER COLUMN id SET DEFAULT nextval('cqc.cqclog_id_seq'::regclass);
```

Comment:
Hi Warren
We might have to use the below statement as I doubt in postgres :: regClass command would work nicely?
Below statement might work if 
ALTER TABLE ONLY cqc."CqcLog" ALTER COLUMN id SET DEFAULT nextval('cqc.cqclog_id_seq');

Reply:
I mean the console may not like ::regClass as far as I have seen so far.
Thanks
Shakir